draft-packages/collapsible/KaizenDraft/Collapsible/Collapsible.tsx

```diff
@@ -97,20 +97,19 @@ export class Collapsible extends React.Component<CollapsibleProps, State> {
         data-automation-id={automationId || `collapsible-container-${id}`}
         {...props} // `title` is missing because it is used for the header; requires breaking change to fix
       >
-        <button
-          type="button"
-          id={buttonId}
+        {/* Disabling these a11y linting errors because there is an IconButton that mitigates these concerns. The onClick here is just an additional layer. */}
+        {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,
+        jsx-a11y/no-static-element-interactions */}
+        <div
           className={classnames(styles.button, {
             [styles.defaultVariant]: open && variant === "default",
             [styles.clearVariant]: open && variant === "clear",
             [styles.sticky]: sticky,
             [styles.open]: open,
           })}
           style={sticky && { top: sticky.top }}
-          onClick={this.handleClick}
-          aria-expanded={open}
-          aria-controls={sectionId}
-          data-automation-id={`collapsible-button-${id}`}
+          onClick={this.handleSectionToggle}
```

Comment:
Just for curiosity, should this div now need a focus state because it's clickable? Or does that have the same effect as having a button in a button?

Reply:
In terms of a keyboard only experience we now have this:
https://user-images.githubusercontent.com/1811583/189813421-dcc542ef-fa93-489b-a6ce-25b554b91503.mov

For the mouse experience, we don't have any focus styling on either the header or the IconButton.